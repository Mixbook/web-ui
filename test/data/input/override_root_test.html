<!doctype html>
<!--
   Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
   for details. All rights reserved. Use of this source code is governed by a
   BSD-style license that can be found in the LICENSE file.
-->
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <script src="packages/web_ui/testing/testing.js"></script>
</head>
<body>
  <element name="x-component" extends="div" constructor="Default">
    <template><div>this should say 'hello': {{topLevelVar}}</div></template>
    <script type='application/dart'>
      import 'common.dart';
      import 'package:web_ui/web_ui.dart';
      class Default extends WebComponent {}
    </script>
  </element>
  <div is="x-component"></div>
  <div id="root">
    <div id="runtime-component"></div>
  </div>
  <script type="application/dart">
    import 'common.dart';
    import 'dart:async';
    import 'dart:html';
    import 'package:unittest/unittest.dart';
    import 'package:web_ui/web_ui.dart';
    main() {
      topLevelVar = 'hello';
      webUIRoot = query("#root");
      Timer.run(() {
        window.postMessage('done', '*');
      });
    }
  </script>
</body>
</html>
