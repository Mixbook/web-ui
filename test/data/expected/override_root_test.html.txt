Content-Type: text/plain
layer at (0,0) size 800x600
  RenderView at (0,0) size 800x600
layer at (0,0) size 800x34
  RenderBlock {HTML} at (0,0) size 800x34
    RenderBody {BODY} at (8,8) size 784x18
      RenderBlock {DIV} at (0,0) size 784x0
      RenderBlock {DIV} at (0,0) size 784x18
        RenderBlock {DIV} at (0,0) size 784x18
          RenderBlock {DIV} at (0,0) size 784x18
            RenderText {#text} at (0,0) size 250x18
              text run at (0,0) width 250: "this should say 'hello': "
            RenderText {#text} at (250,0) size 50x18
              text run at (250,0) width 50: "hello"
